spec/oclc_unset_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative './../lib/lsp-data'
+require 'spec_helper'
+
+
+### Given an OCLC number, a token, and a connection, return a status code and
+###   JSON body
+RSpec.describe LspData::OCLCUnset do
+  subject(:unset) do
+    described_class.new(oclc_num: oclc_num, token: token, conn: conn)
+  end
+
+  context 'Holding is not yet unset' do
+    let(:url) { 'https://metadata.api.oclc.org/worldcat' }
+    let(:conn) { api_conn(url) }
+    let(:oclc_num) { '1' }
+    let(:token) { 'abc' }
+
+    it 'returns message that the holding was unset' do
+      stub_unsend(fixture: 'oclc_unsend_success.json',
```

Comment:
Does `unsend` represent a different concept than `unset`?  It would be nice to choose names that are a bit more distinct if so, or to make the names consistent if not.

Reply:
Good catch! No, it's the same concept. I'll adjust.